develop/vagrant/README.md

```diff
@@ -21,24 +34,181 @@ It is recommended to then install the following VSCode extensions if you don't a
 * CMake Tools
 * Trailing Spaces
 
-Now make any changes and rebuild from the command line:
-### Rebuilding Dobby
+## 4. Connect to Vagrant
 ```
-cd ~/srcDobby/build
-cmake -DCMAKE_BUILD_TYPE=Debug -DRDK_PLATFORM=DEV_VM ../
-make -j$(nproc)
+git clone https://github.com/rdkcentral/DSM.git
+cd DSM/develop/vagrant/
+To connect to the running VM (note: this must be done from the DSM/develop/vagrant/ directory):
+vagrant up
+vagrant ssh
+```
+
+
+### 5. Take vagrant snapshot
```

Comment:
please make sure that all these numbered headings are at the same level. some of them are `##` and some are `###`

Reply:
Addressed.